docs/using-flatpak.rst

```diff
@@ -1,15 +1,54 @@
 Using Flatpak
 =============
 
-This page provides an introduction to the most common commands needed to use Flatpak. It is not intended to be exhaustive or to cover all the options for each command (a full list of all the commands and their options can be found in the :doc:`flatpak-command-reference`.
+This page provides an introduction to the ``flatpak`` command line tool, including the most common commands needed to use Flatpak.
+
+The flatpak command
+--------------------
+
+``flatpak`` is the primary Flatpak command, to which specific commands are appended. For example, the command to install something is ``flatpak install`` and the command to uninstall is ``flatpak uninstall``.
+
+Identifiers
+-----------
+
+Flatpak identifies each application and runtime using a unique identifier, which takes the form of an inverse DNS address, such as ``com.company.App``. The final segment of this address is the object's name, and the preceding part is the domain that it belongs to, which should belong to the developer. Multiple applications can belong to the same domain, such as ``com.company.App1`` and ``com.company.App2``.
```

Comment:
We may want to stop referring to them and "reverse DNS", see also my comment here: https://github.com/flathub/org.filezillaproject.Filezilla/pull/1#issuecomment-354509330

Reply:
I've had a go at addressing this in the latest version of the branch.